developer-docs-site/docs/guides/move-guides/bytecode-dependencies.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Bytecode for Dependencies"
+id: "bytecode-dependencies"
+---
+
+# Use Bytecode for Dependencies when compiling Move
+
+Aptos enables you to use local bytecode as dependencies when the Move source code for those dependencies are not available locally. To use this feature, you will co-locate the files in directories at the same level and then specify their paths in the corresponding `Move.toml` files.
+
+## Requirements and limitations
+
+Using local bytecode as dependencies requires bytecode files to be downloaded into local, and the actual address for each named address must be known and specified in either `Move.toml` or through `--named-addresses`.
+
+The `aptos move prove` and `aptos move test` commands do not support bytecode as dependencies.
+
+## Recommended structure
+
+We use an example to illustrate the dev flow of using this feature. Suppose we want to compile the package `A` . The package layout is:
+
+```rust
+./A
+├── Move.toml 
+├── sources
+  ├ A.move
```

Comment:
Thanks for the advice, Greg! I was thinking of adding an example directly to `move-examples`. However, since the bytecode version will be updated, how should we add this example to make it easy to understand and try?